assemble/bin/accumulo-cluster

```diff
@@ -467,10 +469,17 @@ function control_services() {
     done
   fi
 
-  if [[ $ARG_LOCAL == 0 && $ARG_ALL == 1 && ($operation == "stop" || $operation == "kill") ]]; then
-    if ! isDebug; then
-      echo "Cleaning all server entries in ZooKeeper"
-      $accumulo_cmd org.apache.accumulo.server.util.ZooZap -manager -tservers -compactors -sservers
+  if [[ $ARG_LOCAL == 0 && ($operation == "stop" || $operation == "kill") ]]; then
+    if [[ $ARG_TSERVER == 1 ]]; then
+      if ! isDebug; then
+        echo "Cleaning all tserver entries in ZooKeeper"
+        $accumulo_cmd org.apache.accumulo.server.util.ZooZap -tservers -group "$zap_group"
```

Comment:
If no tserver groups are specified, then `zap_group` will be empty. What happens in this case in ZooZap? Do we also need to check that `zap_group` is not empty?

Reply:
There is an internal check in `ZooZap` that handles an empty group

https://github.com/apache/accumulo/blob/e5370d1d5cd436d86170d16b0e8c0986d3275e15/server/base/src/main/java/org/apache/accumulo/server/util/ZooZap.java#L115-L120